### PR TITLE
Fix #11570 - create gfx_draw_string_centered() overload

### LINKED ISSUE #11570: Use ScreenCoordsXY on gfx_draw_string_centred
Create an overload of `gfx_draw_string_centred` on `\src\openrct2\drawing\Text.cpp` with the same name, but that takes a `const ScreenCoordsXY&` object instead of an `int32_t x` and and an `int32_t y` and replace the calls to the `{x ,y}` pair by the one using `ScreenCoordsXY`.

Don't try to do all calls at once, unless it doesn't occur a lot, as this might lead to a huge PR. When we stop using the old function, we can drop it entirely in favour of the `ScreenCoordsXY` alternative.

### PR DESCRIPTION
**Pull request by TomasZilinek**
I created the overload, updated all calls from the old function to the new and deleted the old one

### CODE REVIEW

File: src/openrct2-ui/windows/ClearScenery.cpp [hunk 1]
```diff
@@ -235,26 +235,28 @@ static void window_clear_scenery_invalidate(rct_window* w)
...
+    ScreenCoordsXY screenCoords;
 
     window_draw_widgets(w, dpi);
 
     // Draw number for tool sizes bigger than 7
-    x = w->windowPos.x
+    screenCoords.x = w->windowPos.x
         + (window_clear_scenery_widgets[WIDX_PREVIEW].left + window_clear_scenery_widgets[WIDX_PREVIEW].right) / 2;
-    y = w->windowPos.y
+    screenCoords.y = w->windowPos.y
```
Comment (tupaschoal) [inline src/openrct2-ui/windows/ClearScenery.cpp:245]: Delay the creation of the variable to here. Something like
```
ScreenCoordsXY screenCoords = ....
```

File: src/openrct2-ui/windows/Footpath.cpp [hunk 2]
```diff
@@ -618,26 +619,27 @@ static void window_footpath_paint(rct_window* w, rct_drawpixelinfo* dpi)
         image += pathType->image;
 
         // Draw construction image
-        int32_t x = w->windowPos.x
+        screenCoords.x = w->windowPos.x
             + (window_footpath_widgets[WIDX_CONSTRUCT].left + window_footpath_widgets[WIDX_CONSTRUCT].right) / 2;
-        int32_t y = w->windowPos.y + window_footpath_widgets[WIDX_CONSTRUCT].bottom - 60;
-        gfx_draw_sprite(dpi, image, x, y, 0);
+        screenCoords.y = w->windowPos.y + window_footpath_widgets[WIDX_CONSTRUCT].bottom - 60;
```
Comment (tupaschoal) [inline src/openrct2-ui/windows/Footpath.cpp:624]: Make these assignments a single statement like
```
screenCoords = { , }
```

(respecting formatting)

File: src/openrct2-ui/windows/Footpath.cpp [hunk 3]
```diff
@@ -618,26 +619,27 @@ static void window_footpath_paint(rct_window* w, rct_drawpixelinfo* dpi)
...
+        gfx_draw_sprite(dpi, image, screenCoords.x, screenCoords.y, 0);
 
         // Draw build this... label
-        x = w->windowPos.x + (window_footpath_widgets[WIDX_CONSTRUCT].left + window_footpath_widgets[WIDX_CONSTRUCT].right) / 2;
-        y = w->windowPos.y + window_footpath_widgets[WIDX_CONSTRUCT].bottom - 23;
-        gfx_draw_string_centred(dpi, STR_BUILD_THIS, x, y, COLOUR_BLACK, nullptr);
+        screenCoords.x = w->windowPos.x
+            + (window_footpath_widgets[WIDX_CONSTRUCT].left + window_footpath_widgets[WIDX_CONSTRUCT].right) / 2;
+        screenCoords.y = w->windowPos.y + window_footpath_widgets[WIDX_CONSTRUCT].bottom - 23;
```
Comment (tupaschoal) [inline src/openrct2-ui/windows/Footpath.cpp:630]: Make these assignments a single statement like
```
screenCoords = { , }
```

(respecting formatting)

File: src/openrct2-ui/windows/Footpath.cpp [hunk 4]
```diff
@@ -618,26 +619,27 @@ static void window_footpath_paint(rct_window* w, rct_drawpixelinfo* dpi)
...
+        gfx_draw_string_centred(dpi, STR_BUILD_THIS, screenCoords, COLOUR_BLACK, nullptr);
     }
 
     // Draw cost
-    int32_t x = w->windowPos.x
+    screenCoords.x = w->windowPos.x
         + (window_footpath_widgets[WIDX_CONSTRUCT].left + window_footpath_widgets[WIDX_CONSTRUCT].right) / 2;
-    int32_t y = w->windowPos.y + window_footpath_widgets[WIDX_CONSTRUCT].bottom - 12;
+    screenCoords.y = w->windowPos.y + window_footpath_widgets[WIDX_CONSTRUCT].bottom - 12;
```
Comment (tupaschoal) [inline src/openrct2-ui/windows/Footpath.cpp:637]: Make these assignments a single statement like
```
screenCoords = { , }
```

(respecting formatting)

File: src/openrct2-ui/windows/GameBottomToolbar.cpp [hunk 5]
```diff
@@ -531,8 +531,8 @@ static void window_game_bottom_toolbar_draw_right_panel(rct_drawpixelinfo* dpi,
     uint32_t line_height = font_get_line_height(FONT_SPRITE_BASE_MEDIUM);
 
     // Temperature
-    x = w->windowPos.x + window_game_bottom_toolbar_widgets[WIDX_RIGHT_OUTSET].left + 15;
-    y += line_height + 1;
+    screenCoords.x = w->windowPos.x + window_game_bottom_toolbar_widgets[WIDX_RIGHT_OUTSET].left + 15;
+    screenCoords.y += line_height + 1;
```
Comment (tupaschoal) [inline src/openrct2-ui/windows/GameBottomToolbar.cpp:535]: Make these assignments a single statement like
```
screenCoords = { , }
```

(respecting formatting)

File: src/openrct2-ui/windows/Land.cpp [hunk 6]
```diff
@@ -348,34 +349,35 @@ static void window_land_paint(rct_window* w, rct_drawpixelinfo* dpi)
     // Draw number for tool sizes bigger than 7
     if (gLandToolSize > MAX_TOOL_SIZE_WITH_SPRITE)
     {
-        x = w->windowPos.x + (previewWidget->left + previewWidget->right) / 2;
-        y = w->windowPos.y + (previewWidget->top + previewWidget->bottom) / 2;
-        gfx_draw_string_centred(dpi, STR_LAND_TOOL_SIZE_VALUE, x, y - 2, COLOUR_BLACK, &gLandToolSize);
+        screenCoords.x = w->windowPos.x + (previewWidget->left + previewWidget->right) / 2;
+        screenCoords.y = w->windowPos.y + (previewWidget->top + previewWidget->bottom) / 2;
```
Comment (tupaschoal) [inline src/openrct2-ui/windows/Land.cpp:353]: Make these assignments a single statement like
```
screenCoords = { , }
```

(respecting formatting)

File: src/openrct2-ui/windows/Land.cpp [hunk 7]
```diff
@@ -348,34 +349,35 @@ static void window_land_paint(rct_window* w, rct_drawpixelinfo* dpi)
...
+        gfx_draw_string_centred(
+            dpi, STR_LAND_TOOL_SIZE_VALUE, screenCoords - ScreenCoordsXY{ 0, 2 }, COLOUR_BLACK, &gLandToolSize);
     }
     else if (gLandMountainMode)
     {
-        x = w->windowPos.x + previewWidget->left;
-        y = w->windowPos.y + previewWidget->top;
+        screenCoords.x = w->windowPos.x + previewWidget->left;
+        screenCoords.y = w->windowPos.y + previewWidget->top;
```
Comment (tupaschoal) [inline src/openrct2-ui/windows/Land.cpp:360]: Make these assignments a single statement like
```
screenCoords = { , }
```

(respecting formatting)

File: src/openrct2-ui/windows/Land.cpp [hunk 8]
```diff
@@ -348,34 +349,35 @@ static void window_land_paint(rct_window* w, rct_drawpixelinfo* dpi)
...
-        gfx_draw_sprite(dpi, sprite, x, y, 0);
+        gfx_draw_sprite(dpi, sprite, screenCoords.x, screenCoords.y, 0);
         widget_draw(dpi, w, WIDX_DECREMENT);
         widget_draw(dpi, w, WIDX_INCREMENT);
     }
 
-    x = w->windowPos.x + (previewWidget->left + previewWidget->right) / 2;
-    y = w->windowPos.y + previewWidget->bottom + 5;
+    screenCoords.x = w->windowPos.x + (previewWidget->left + previewWidget->right) / 2;
+    screenCoords.y = w->windowPos.y + previewWidget->bottom + 5;
```
Comment (tupaschoal) [inline src/openrct2-ui/windows/Land.cpp:368]: Make these assignments a single statement like
```
screenCoords = { , }
```

(respecting formatting)

File: src/openrct2-ui/windows/LandRights.cpp [hunk 9]
```diff
@@ -271,25 +271,28 @@ static void window_land_rights_invalidate(rct_window* w)
...
 {
-    int32_t x, y;
+    ScreenCoordsXY screenCoords;
 
-    x = w->windowPos.x + (window_land_rights_widgets[WIDX_PREVIEW].left + window_land_rights_widgets[WIDX_PREVIEW].right) / 2;
-    y = w->windowPos.y + (window_land_rights_widgets[WIDX_PREVIEW].top + window_land_rights_widgets[WIDX_PREVIEW].bottom) / 2;
+    screenCoords.x = w->windowPos.x
+        + (window_land_rights_widgets[WIDX_PREVIEW].left + window_land_rights_widgets[WIDX_PREVIEW].right) / 2;
+    screenCoords.y = w->windowPos.y
+        + (window_land_rights_widgets[WIDX_PREVIEW].top + window_land_rights_widgets[WIDX_PREVIEW].bottom) / 2;
```
Comment (tupaschoal) [inline src/openrct2-ui/windows/LandRights.cpp:279]: Make these assignments a single statement like
```
screenCoords = { , }
```
Also move the declaration of the variable to this same line
(respecting formatting)

File: src/openrct2-ui/windows/LandRights.cpp [hunk 10]
```diff
@@ -271,25 +271,28 @@ static void window_land_rights_invalidate(rct_window* w)
...
 
     // Draw cost amount
     if (_landRightsCost != MONEY32_UNDEFINED && _landRightsCost != 0 && !(gParkFlags & PARK_FLAGS_NO_MONEY))
     {
-        x = (window_land_rights_widgets[WIDX_PREVIEW].left + window_land_rights_widgets[WIDX_PREVIEW].right) / 2
+        screenCoords.x = (window_land_rights_widgets[WIDX_PREVIEW].left + window_land_rights_widgets[WIDX_PREVIEW].right) / 2
             + w->windowPos.x;
-        y = window_land_rights_widgets[WIDX_PREVIEW].bottom + w->windowPos.y + 32;
-        gfx_draw_string_centred(dpi, STR_COST_AMOUNT, x, y, COLOUR_BLACK, &_landRightsCost);
+        screenCoords.y = window_land_rights_widgets[WIDX_PREVIEW].bottom + w->windowPos.y + 32;
```
Comment (tupaschoal) [inline src/openrct2-ui/windows/LandRights.cpp:294]: Make these assignments a single statement like
```
screenCoords = { , }
```

(respecting formatting)

File: src/openrct2-ui/windows/Map.cpp [hunk 11]
```diff
@@ -814,33 +814,36 @@ static void window_map_paint(rct_window* w, rct_drawpixelinfo* dpi)
...
-    y = w->windowPos.y + window_map_widgets[WIDX_LAND_TOOL].bottom + 5;
+    screenCoords.y = w->windowPos.y + window_map_widgets[WIDX_LAND_TOOL].bottom + 5;
 
     // People starting position (scenario editor only)
     if (w->widgets[WIDX_PEOPLE_STARTING_POSITION].type != WWT_EMPTY)
     {
-        x = w->windowPos.x + w->widgets[WIDX_PEOPLE_STARTING_POSITION].left + 12;
-        y = w->windowPos.y + w->widgets[WIDX_PEOPLE_STARTING_POSITION].top + 18;
+        screenCoords.x = w->windowPos.x + w->widgets[WIDX_PEOPLE_STARTING_POSITION].left + 12;
+        screenCoords.y = w->windowPos.y + w->widgets[WIDX_PEOPLE_STARTING_POSITION].top + 18;
```
Comment (tupaschoal) [inline src/openrct2-ui/windows/Map.cpp:834]: Make these assignments a single statement like
```
screenCoords = { , }
```

(respecting formatting)

File: src/openrct2-ui/windows/Map.cpp [hunk 12]
```diff
@@ -814,33 +814,36 @@ static void window_map_paint(rct_window* w, rct_drawpixelinfo* dpi)
...
 
     if (!(gScreenFlags & SCREEN_FLAGS_SCENARIO_EDITOR) && !gCheatsSandboxMode)
     {
         // Render the map legend
         if (w->selected_tab == PAGE_RIDES)
         {
-            x = w->windowPos.x + 4;
-            y = w->windowPos.y + w->widgets[WIDX_MAP].bottom + 2;
+            screenCoords.x = w->windowPos.x + 4;
+            screenCoords.y = w->windowPos.y + w->widgets[WIDX_MAP].bottom + 2;
```
Comment (tupaschoal) [inline src/openrct2-ui/windows/Map.cpp:846]: Make these assignments a single statement like
```
screenCoords = { , }
```

(respecting formatting)

File: src/openrct2-ui/windows/Ride.cpp [hunk 13]
```diff
@@ -2920,8 +2920,9 @@ static void window_ride_main_paint(rct_window* w, rct_drawpixelinfo* dpi)
 
     widget = &window_ride_main_widgets[WIDX_VIEW];
     gfx_draw_string_centred(
-        dpi, STR_WINDOW_COLOUR_2_STRINGID, w->windowPos.x + (widget->left + widget->right - 11) / 2,
-        w->windowPos.y + widget->top, COLOUR_BLACK, gCommonFormatArgs);
+        dpi, STR_WINDOW_COLOUR_2_STRINGID,
+        ScreenCoordsXY{ w->windowPos.x + (widget->left + widget->right - 11) / 2, w->windowPos.y + widget->top }, COLOUR_BLACK,
```
Comment (tupaschoal) [inline src/openrct2-ui/windows/Ride.cpp:2924]:
```suggestion
        { w->windowPos.x + (widget->left + widget->right - 11) / 2, w->windowPos.y + widget->top }, COLOUR_BLACK,
```

File: src/openrct2-ui/windows/RideConstruction.cpp [hunk 14]
```diff
@@ -2329,26 +2330,26 @@ static void window_ride_construction_paint(rct_window* w, rct_drawpixelinfo* dpi
         return;
 
     // Draw track piece
-    x = w->windowPos.x + widget->left + 1;
-    y = w->windowPos.y + widget->top + 1;
+    screenCoords.x = w->windowPos.x + widget->left + 1;
+    screenCoords.y = w->windowPos.y + widget->top + 1;
```
Comment (tupaschoal) [inline src/openrct2-ui/windows/RideConstruction.cpp:2334]: Make these assignments a single statement like
```
screenCoords = { , }
```
Delay the variable creation to here
(respecting formatting)

File: src/openrct2-ui/windows/RideConstruction.cpp [hunk 15]
```diff
@@ -2329,26 +2330,26 @@ static void window_ride_construction_paint(rct_window* w, rct_drawpixelinfo* dpi
...
     {
         window_ride_construction_draw_track_piece(
             w, &clipdpi, rideIndex, trackType, trackDirection, liftHillAndInvertedState, width, height);
     }
 
     // Draw cost
-    x = w->windowPos.x + (widget->left + widget->right) / 2;
-    y = w->windowPos.y + widget->bottom - 23;
+    screenCoords.x = w->windowPos.x + (widget->left + widget->right) / 2;
+    screenCoords.y = w->windowPos.y + widget->bottom - 23;
```
Comment (tupaschoal) [inline src/openrct2-ui/windows/RideConstruction.cpp:2345]: Make these assignments a single statement like
```
screenCoords = { , }
```

(respecting formatting)

File: src/openrct2-ui/windows/TitleScenarioSelect.cpp [hunk 16]
```diff
@@ -604,7 +604,7 @@ static void window_scenarioselect_scrollpaint(rct_window* w, rct_drawpixelinfo*
                 {
                     gCurrentFontSpriteBase = FONT_SPRITE_BASE_MEDIUM_DARK;
                 }
-                gfx_draw_string_centred(dpi, format, wide ? 270 : 210, y + 1, colour, gCommonFormatArgs);
+                gfx_draw_string_centred(dpi, format, ScreenCoordsXY{ wide ? 270 : 210, y + 1 }, colour, gCommonFormatArgs);
```
Comment (tupaschoal) [inline src/openrct2-ui/windows/TitleScenarioSelect.cpp:607]:
```suggestion
                gfx_draw_string_centred(dpi, format, { wide ? 270 : 210, y + 1 }, colour, gCommonFormatArgs);
```

File: src/openrct2-ui/windows/TitleScenarioSelect.cpp [hunk 17]
```diff
@@ -643,7 +643,7 @@ static void draw_category_heading(
 
     // Draw string
     int32_t centreX = (left + right) / 2;
-    gfx_draw_string_centred(dpi, stringId, centreX, y, baseColour, nullptr);
+    gfx_draw_string_centred(dpi, stringId, ScreenCoordsXY{ centreX, y }, baseColour, nullptr);
```
Comment (tupaschoal) [inline src/openrct2-ui/windows/TitleScenarioSelect.cpp:646]:
```suggestion
    gfx_draw_string_centred(dpi, stringId, { centreX, y }, baseColour, nullptr);
```

File: src/openrct2-ui/windows/Water.cpp [hunk 18]
```diff
@@ -211,29 +211,31 @@ static void window_water_invalidate(rct_window* w)
  */
 static void window_water_paint(rct_window* w, rct_drawpixelinfo* dpi)
 {
-    int32_t x, y;
+    ScreenCoordsXY screenCoords;
 
-    x = w->windowPos.x + (window_water_widgets[WIDX_PREVIEW].left + window_water_widgets[WIDX_PREVIEW].right) / 2;
-    y = w->windowPos.y + (window_water_widgets[WIDX_PREVIEW].top + window_water_widgets[WIDX_PREVIEW].bottom) / 2;
+    screenCoords.x = w->windowPos.x + (window_water_widgets[WIDX_PREVIEW].left + window_water_widgets[WIDX_PREVIEW].right) / 2;
+    screenCoords.y = w->windowPos.y + (window_water_widgets[WIDX_PREVIEW].top + window_water_widgets[WIDX_PREVIEW].bottom) / 2;
```
Comment (tupaschoal) [inline src/openrct2-ui/windows/Water.cpp:217]: Make these assignments a single statement like
```
screenCoords = { , }
```
Also delay the variable declaration to here.
(respecting formatting)

File: src/openrct2-ui/windows/Water.cpp [hunk 19]
```diff
@@ -211,29 +211,31 @@ static void window_water_invalidate(rct_window* w)
...
     }
 
     if (!(gParkFlags & PARK_FLAGS_NO_MONEY))
     {
         // Draw raise cost amount
-        x = (window_water_widgets[WIDX_PREVIEW].left + window_water_widgets[WIDX_PREVIEW].right) / 2 + w->windowPos.x;
-        y = window_water_widgets[WIDX_PREVIEW].bottom + w->windowPos.y + 5;
+        screenCoords.x = (window_water_widgets[WIDX_PREVIEW].left + window_water_widgets[WIDX_PREVIEW].right) / 2
+            + w->windowPos.x;
+        screenCoords.y = window_water_widgets[WIDX_PREVIEW].bottom + w->windowPos.y + 5;
```
Comment (tupaschoal) [inline src/openrct2-ui/windows/Water.cpp:232]: Make these assignments a single statement like
```
screenCoords = { , }
```

(respecting formatting)

File: src/openrct2-ui/windows/EditorBottomToolbar.cpp [hunk 20]
```diff
@@ -485,8 +485,8 @@ void window_editor_bottom_toolbar_paint(rct_window* w, rct_drawpixelinfo* dpi)
             if (gScreenFlags & SCREEN_FLAGS_TRACK_DESIGNER)
                 stringId = STR_EDITOR_STEP_OBJECT_SELECTION;
 
-            gfx_draw_string_centred(dpi, STR_BACK_TO_PREVIOUS_STEP, textX, textY, textColour, nullptr);
-            gfx_draw_string_centred(dpi, stringId, textX, textY + 10, textColour, nullptr);
+            gfx_draw_string_centred(dpi, STR_BACK_TO_PREVIOUS_STEP, ScreenCoordsXY{ textX, textY }, textColour, nullptr);
```
Comment (tupaschoal) [inline src/openrct2-ui/windows/EditorBottomToolbar.cpp:488]:
```suggestion
            gfx_draw_string_centred(dpi, STR_BACK_TO_PREVIOUS_STEP, { textX, textY }, textColour, nullptr);
```

File: src/openrct2-ui/windows/EditorBottomToolbar.cpp [hunk 21]
```diff
@@ -485,8 +485,8 @@ void window_editor_bottom_toolbar_paint(rct_window* w, rct_drawpixelinfo* dpi)
...
+            gfx_draw_string_centred(dpi, stringId, ScreenCoordsXY{ textX, textY + 10 }, textColour, nullptr);
```
Comment (tupaschoal) [inline src/openrct2-ui/windows/EditorBottomToolbar.cpp:489]:
```suggestion
            gfx_draw_string_centred(dpi, stringId, { textX, textY + 10 }, textColour, nullptr);
```

File: src/openrct2-ui/windows/EditorBottomToolbar.cpp [hunk 22]
```diff
@@ -512,8 +512,8 @@ void window_editor_bottom_toolbar_paint(rct_window* w, rct_drawpixelinfo* dpi)
             if (gScreenFlags & SCREEN_FLAGS_TRACK_DESIGNER)
                 stringId = STR_EDITOR_STEP_ROLLERCOASTER_DESIGNER;
 
-            gfx_draw_string_centred(dpi, STR_FORWARD_TO_NEXT_STEP, textX, textY, textColour, nullptr);
-            gfx_draw_string_centred(dpi, stringId, textX, textY + 10, textColour, nullptr);
+            gfx_draw_string_centred(dpi, STR_FORWARD_TO_NEXT_STEP, ScreenCoordsXY{ textX, textY }, textColour, nullptr);
```
Comment (tupaschoal) [inline src/openrct2-ui/windows/EditorBottomToolbar.cpp:515]:
```suggestion
            gfx_draw_string_centred(dpi, STR_FORWARD_TO_NEXT_STEP, { textX, textY }, textColour, nullptr);
```

File: src/openrct2-ui/windows/EditorBottomToolbar.cpp [hunk 23]
```diff
@@ -512,8 +512,8 @@ void window_editor_bottom_toolbar_paint(rct_window* w, rct_drawpixelinfo* dpi)
...
+            gfx_draw_string_centred(dpi, stringId, ScreenCoordsXY{ textX, textY + 10 }, textColour, nullptr);
```
Comment (tupaschoal) [inline src/openrct2-ui/windows/EditorBottomToolbar.cpp:516]:
```suggestion
            gfx_draw_string_centred(dpi, stringId, { textX, textY + 10 }, textColour, nullptr);
```

File: src/openrct2-ui/windows/LandRights.cpp [hunk 24]
```diff
@@ -271,25 +271,28 @@ static void window_land_rights_invalidate(rct_window* w)
 
 static void window_land_rights_paint(rct_window* w, rct_drawpixelinfo* dpi)
 {
-    int32_t x, y;
+    ScreenCoordsXY screenCoords;
 
-    x = w->windowPos.x + (window_land_rights_widgets[WIDX_PREVIEW].left + window_land_rights_widgets[WIDX_PREVIEW].right) / 2;
-    y = w->windowPos.y + (window_land_rights_widgets[WIDX_PREVIEW].top + window_land_rights_widgets[WIDX_PREVIEW].bottom) / 2;
+    screenCoords = {
```
Comment (tupaschoal) [inline src/openrct2-ui/windows/LandRights.cpp:276]: Declare the variable and initialize it in the same statement, like you did before: `auto screenCoords = ScreenCoordsXY{...}`

File: src/openrct2-ui/windows/RideConstruction.cpp [hunk 25]
```diff
@@ -2329,26 +2330,24 @@ static void window_ride_construction_paint(rct_window* w, rct_drawpixelinfo* dpi
         return;
 
     // Draw track piece
-    x = w->windowPos.x + widget->left + 1;
-    y = w->windowPos.y + widget->top + 1;
+    screenCoords = { w->windowPos.x + widget->left + 1, w->windowPos.y + widget->top + 1 };
```
Comment (tupaschoal) [inline src/openrct2-ui/windows/RideConstruction.cpp:2333]: Declare and initialize the variable here instead of up there.

**Comment by TomasZilinek** [comment]
Ook, I am sorry. I will correct all of them this time :D

**Pull request merged**